ios/docs/help/about/system-requirements.md

```diff
@@ -6,4 +6,4 @@ title: System Requirements - Keyman for iPhone and iPad Help
 
 ### Minimum iOS Version
 Keyman for iPhone and iPad will run on Apple iPhones and iPads that have a minimum version of
-iOS 12.1.
+iOS 15.6
```

Comment:
Can you also update:
* https://github.com/keymanapp/keyman/blob/master/resources/build/minimum-versions.inc.sh
* https://github.com/keymanapp/keyman/wiki/Keyman-18.0-Release-Notes

Reply:
fyi I ran the `publish-minimum-versions.sh` script which auto-generates minimum-versions.md too.

Reply:
@darcywong00 So I guess I shouldn't be modifying minimum-versions.md directly? I'm not familiar with the process for that.

Reply:
Yeah, the `publish-minimum-versions.sh` script updates the .md from the .inc.sh, just helps to keep them in sync.

Note that the .md says this already:

```
> ### This file is auto-generated by publish-minimum-versions.sh. Do not modify
```